### ISSUE TITLE
Accessibility: Not able to navigate to the view bar options  in the "vs code editor" using "Tab" key for the first time.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17264]
VS Code Version: 1.6.1
OS Version:1607

**Steps to Reproduce:**
1.Open visual studio code editor.
2.Using "Tab", key try to navigate to the "viewbaroptions (Explorer, Search, Git, Debug and Extensions)"

**Expected result:**
User should be able to navigate to all the "view bar options" using "Tab" key for the first time.

**Actual Result:**
User is not able to navigate to the view bar options using "Tab", key.User has to travel a lot using "Tab",key to reach view bar options. 

**User Impact:**
User might wouldn't able to Navigate to View Bar without using short cut keys for the first time.

**Attachments:**
![vscodeditor_viewbaroptions](https://cloud.githubusercontent.com/assets/22956130/21215261/16785f3c-c2c6-11e6-84cb-4cf138172287.jpg)

**Comment by bpasero** [comment]
@WiproPSAccTest that depends on where the keyboard focus is when you start. I think in your case it was inside the editor and then you need more keys to go there. Can you find out where you started from?

**Comment by bpasero** [comment]
Timeout.

**Comment by WiproPSAccTest** [comment]
Title has been changed to a common format to perform data population.
